src/components/CurrencyInputPanel/currency-panel.scss

```diff
@@ -10,6 +10,16 @@
     border: 1px solid $mercury-gray;
     background-color: $white;
     box-shadow: 0px 4px 4px 2px rgba($royal-blue, 0.05);
+
+    &--error {
+      border: 0.5px solid $error-red;
+
+      .currency-input-panel {
+        &__input, &__extra-text {
+          color: $error-red;
```

Comment:
I see potential problems for always including extra-text as part of error state. Insufficient balance is our only error state right now, but in the future we might have other error states that doesn't have anything to do with extra text.

One case i can think of is in the Output panel, the user won't get into an insufficient balance state, but they will have an error state when Output value is higher then Output reserve, causing the exchange rate to go negative. 

My suggestion would be:
- Create an ERROR constants like `{ INSUFFICIENT_BALANCE: 'Insufficient balance' }`
- We will have a react or redux state `errorType`
- based on error type, we will apply the error to different components

```jsx
<div
    className={classnames('default', {
        default__error: this.state.error === INSUFFICIENT_BALANCE,
    })}
/>
```

Reply:
Alright makes sense. Just updated this PR to include logic for error handling within CurrencyInputPanel (that connects with redux. Let me know what you think. 

We need to connect it with redux so that the parent component will be able to disable CTA if there are errors for input/output.